### ISSUE TITLE
Improve loading styles

### ISSUE THREAD

**Issue by cobraz** [bjerkio/website#98]
We should probably add some prettier loading. Preloading (or something)?

Check this video to see the loading without local cache: http://www.youtube.com/watch?v=J-PXNNwprTg

/cc @AnneMatilde @dkostenko-bjerk

**Comment by AnneMatilde** [comment]
Yes, I agree. How do we solve this? Do we need a specific design or can we use something else?

**Comment by cobraz** [comment]
> We can add a Spinner component, but how do we track the event that the site is fully loaded?

I don't know. We can probably preload fonts at least? Let me do some research on it :)

**Comment by AnneMatilde** [comment]
Maybe I can design a greyscale "skeleton" as a preload animation?

**Comment by cobraz** [comment]
That would be awesome! 👍

**Comment by AnneMatilde** [comment]
What about something like this? @cobraz 

![image](https://user-images.githubusercontent.com/54155334/112721254-34e48f00-8f03-11eb-8854-7c18e6c2fd00.png)

**Comment by dkostenko-bjerk** [comment]
it’s good, but I think we must create “skeleton” for all pages

**Comment by AnnaBjerk** [comment]
> it’s good, but I think we should create “skeleton” for all pages

What is your opinion on that, @AnneMatilde 

/cc: @cobraz

**Comment by AnneMatilde** [comment]
It would look nicer, but I'm not sure it is needed. @cobraz what do you think?

**Comment by AnnaBjerk** [comment]
Indeed. Dmitry and I discussed it during daily today. It seems that the only place the skeleton might be needed is the home page because of the video loading slowly. The other pages don't really have any loading time. They pop up rather quickly. Curious about @cobraz thoughts on this.
